src/patternfly/components/ChipGroup/examples/chip-group-toolbar-expanded-example.hbs

```diff
@@ -39,21 +43,24 @@
     {{#> chip-text chip-text--attribute=(concat 'id="' chip-group--id 'chip_five"')}}
       Chip One
     {{/chip-text}}
-    {{#> button button--modifier="pf-m-plain" button--HasTimesCircleIcon="true" button--attribute=(concat 'aria-labelledby="' chip-group--id 'remove_chip_five ' chip-group--id 'chip_five" aria-label="Remove" id="' chip-group--id 'remove_chip_five"')}}
+    {{#> button button--modifier="pf-m-plain" button--attribute=(concat 'aria-labelledby="' chip-group--id 'remove_chip_five ' chip-group--id 'chip_five" aria-label="Remove" id="' chip-group--id 'remove_chip_five"')}}
+      {{> button-icon button-icon--type="times-circle"}}
     {{/button}}
   {{/chip}}
   {{#> chip}}
     {{#> chip-text chip-text--attribute=(concat 'id="' chip-group--id 'chip_six"')}}
       Chip Two
     {{/chip-text}}
-    {{#> button button--modifier="pf-m-plain" button--HasTimesCircleIcon="true" button--attribute=(concat 'aria-labelledby="' chip-group--id 'remove_chip_six ' chip-group--id 'chip_six" aria-label="Remove" id="' chip-group--id 'remove_chip_six"')}}
+    {{#> button button--modifier="pf-m-plain"  button--attribute=(concat 'aria-labelledby="' chip-group--id 'remove_chip_six ' chip-group--id 'chip_six" aria-label="Remove" id="' chip-group--id 'remove_chip_six"')}}
```

Comment:
remove the extra space before `button--attribute`?

Reply:
spacing adjusted.